org.sebicodes.Prt.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.sebicodes.Prt
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+tag: "1.0.0"
+commit": "d60a7d3ee075d4c74f6a8120f68fb689e48ea419"
+command: runner.sh
+modules:
+  - name: app
+    buildsystem: simple
+    build-commands:
+      - install -D runner.sh /app/bin/runner.sh
+      - install -D app.py /app/bin/app.py
+      - install -D .desktop /app/share/applications/com.sebicodes.Prt.desktop
+      - install -D plruth.png /app/share/icons/hicolor/500x500/apps/com.sebicodes.Prt.png
```

Comment:
500x500 is not a valid icon size.

Reply:
also the name doesn't match the id.

Reply:
Hello, thanks for your revision. What should be a valid size?
I will fix it now